src/main/io/serial_cli.c

```diff
@@ -1968,30 +1960,6 @@ static void cliStatus(char *cmdline)
     printf("Cycle Time: %d, I2C Errors: %d, config size: %d\r\n", cycleTime, i2cErrorCounter, sizeof(master_t));
 }
 
-#ifdef USE_SERIAL_1WIRE
-static void cliUSB1Wire(char *cmdline)
```

Comment:
i think we may want to keep this, even though the configurator doesn't use it, it is useful for testing and single esc configuration